docker-compose.yaml

```
@@ -24,6 +24,12 @@ services:
         SPDK_GIT_COMMIT:
       labels:
         io.ceph.nvmeof:
+  # Used to export RPM packages externally (via bind mount)
+  spdk-rpm-export:
+    extends:
```

```suggestion
    extends:
    image: spdk-rpm-export
```

@epuertat, noticed CI test does not export RPMs, see [Upload stand-alone packages step](https://github.com/ceph/ceph-nvmeof/actions/runs/5741111843/job/15560476570?pr=169):
```
With the provided path, there will be 1 file uploaded
```
It seems the `spdk-rpm-export` service defaulting to `spdk` as image name.

A possible solution is to give the service explicit image name. Without explicit build during `Build container images`, the container image is created during `make export-rpms`

---

Argh, you're right! Let me change that. Thanks for the troubleshooting :D